tests/integration/models_services_test.py

```diff
@@ -203,6 +205,50 @@ def test_scale_service(self):
         spec = service.attrs['Spec']['TaskTemplate']['ContainerSpec']
         assert spec.get('Command') == ['sleep', '300']
 
+    def test_scale_method_service(self):
```

Comment:
This test is failing because the default `mode` for a service is `global`. Make sure to set the initial mode to `replicated`!

Reply:
Thanks @shin-. I already changed.